modRlp/src/org/aion/rlp/RLP.java

```diff
@@ -647,23 +458,24 @@ public static DecodeResult decode(byte[] data, int pos) {
         if (prefix == OFFSET_SHORT_ITEM) {
             return new DecodeResult(pos + 1, ""); // means no length or 0
         } else if (prefix < OFFSET_SHORT_ITEM) {
-            return new DecodeResult(pos + 1, new byte[] { data[pos] }); // byte
-                                                                        // is
-                                                                        // its
-                                                                        // own
-                                                                        // RLP
-                                                                        // encoding
-        } else if (prefix < OFFSET_LONG_ITEM) {
+            return new DecodeResult(pos + 1, new byte[] {data[pos]}); // byte
+            // is
+            // its
+            // own
+            // RLP
```

Comment:
awkward

Reply:
fixed in PR #572 commit ba5c2954639c95eee8258f5d9c4392d1ad147225